Rust/Symbol List.tmPreferences

```diff
@@ -9,7 +9,8 @@
 		entity.name.module.rust,
 		entity.name.struct.rust,
 		entity.name.trait.rust,
-		entity.name.union.rust
+		entity.name.union.rust,
+        meta.enum.rust storage.type - meta.struct
```

Comment:
This line uses spaces for indentation while the rest of the file uses tabs.

Can you also add a "negative" test for the `meta.struct` case?

Reply:
fixed and done, thanks :)